src/OrchardCore/OrchardCore.DisplayManagement.Liquid/RequestLiquidTemplateEventHandler.cs

```diff
@@ -117,5 +120,15 @@ public HttpContextItemsWrapper(IDictionary<object,object> items)
                 Items = items;
             }
         }
+
+        private class RouteValueDictionaryWrapper
+        {
+            public readonly IReadOnlyDictionary<string, object> RouteValueDictionary;
```

Comment:
Why it's setter?

Reply:
@hishamco I don't understand your question. Can you rephrase please?